.github/workflows/CI.yml

```diff
@@ -1,3 +1,5 @@
+env:
+  JULIA_NUM_THREADS: 4
```

Comment:
@ffevotte With this we can run the CI using multiple threads. Assuming the CI machine has multiple cores, perhaps the figures be directly included from the CI for all but the `README.md` pages in the documentation?

Reply:
Ok, so we better make sure that there are enough cores for the threads, or some of the tests will fail. It seems that the GitHub runners have two cores, so I am changing this to 2 threads. 

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources